src/init.cpp

```diff
@@ -555,7 +555,7 @@ std::string HelpMessage(HelpMessageMode mode)
     strUsage += HelpMessageOpt("-timeout=<n>", strprintf(_("Specify connection timeout in milliseconds (minimum: 1, default: %d)"), DEFAULT_CONNECT_TIMEOUT));
     strUsage += HelpMessageOpt("-torcontrol=<ip>:<port>", strprintf(_("Tor control port to use if onion listening enabled (default: %s)"), DEFAULT_TOR_CONTROL));
     strUsage += HelpMessageOpt("-torpassword=<pass>", _("Tor control port password (default: empty)"));
-    strUsage += HelpMessageOpt("-asmap=<file>", _("Specify asn mapping used for bucketing of the peers. Path should be relative to the -datadir path."));
+    strUsage += HelpMessageOpt("-asmap=<file>", strprintf(_("Specify asn mapping used for bucketing of the peers (default: %s). Path should be relative to the -datadir path."), DEFAULT_ASMAP_FILENAME));
```

Comment:
pls move it above `-banscore` like in the original PR to keep cmd line options sorted properly

Reply:
Resolved in https://github.com/dashpay/dash/pull/4028/commits/6e1a7318a715c61ae1b737ed19e18cad35448f17